Demo/DemoApp/Cartfile.resolved

```diff
@@ -0,0 +1 @@
+github "google/EarlGrey" "ad70b511da47793e27e77c1a69c7895e8403be45"
```

Comment:
Pinned to a commit?

Reply:
`carthage update` will refresh the `Cartfile.resolved` file with the latest commit

`Cartfile` is listed as tracking earlgrey2 branch